go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/aws/karpenter-provider-aws
 go 1.22
 
 require (
+	dario.cat/mergo v1.0.0
```

Comment:
What's this new dep and why do we need it?

Reply:
Got that confused with `github.com/imdario/mergo`